dtests/src/test/java/io/snappydata/hydra/smoke.sh

```diff
@@ -43,4 +43,7 @@ sleep 30;
 $SNAPPYDATA_SOURCE_DIR/store/tests/core/src/main/java/bin/sample-runbt.sh $resultDir $SNAPPYDATA_SOURCE_DIR  -r 1  -d false io/snappydata/hydra/smoke.bt
 sleep 30;
 
+$SNAPPYDATA_SOURCE_DIR/store/tests/core/src/main/java/bin/sample-runbt.sh $resultDir $SNAPPYDATA_SOURCE_DIR  -r 1  -d false io/snappydata/hydra/smokePerf.bt
```

Comment:
As the smokePerf run is not using the snappy cluster started using start startDualModeCluster.conf run in smoke bt, you can move the smokePerf run in the beginning. smoke bt will be then run after smokePerf run.